web_src/js/features/repo-issue-list.js

```diff
@@ -119,8 +120,80 @@ function initRepoIssueListAuthorDropdown() {
   };
 }
 
+function initPinRemoveButton() {
+  for (const button of document.getElementsByName('issue-unpin-button')) {
+    button.addEventListener('click', (event) => {
+      const elem = event.currentTarget;
+      const id = Number(elem.getAttribute('data-issue-id'));
+
+      // Send the unpin request
+      $.ajax({
+        url: elem.getAttribute('data-unpin-url'),
+        headers: {
+          'X-Csrf-Token': window.config.csrfToken,
+        },
+        contentType: 'application/json',
+        type: 'DELETE',
+        error: (xhr) => {
+          throw new Error(xhr.responseText);
+        },
+        success: () => {
+          // Remove the Card
+          document.querySelector(`div[data-issue-id="${id}"]`).remove();
+        }
+      });
+    });
+  }
+}
+
+function pinMoveEnd(event) {
+  const url = event.item.getAttribute('data-move-url');
+  const id = Number(event.item.getAttribute('data-issue-id'));
+  $.ajax({
+    url,
+    data: JSON.stringify({id, position: event.newIndex + 1}),
+    headers: {
+      'X-Csrf-Token': window.config.csrfToken,
+    },
+    contentType: 'application/json',
+    type: 'POST',
+    error: (xhr) => {
+      throw new Error(xhr.responseText);
+    }
+  });
```

Comment:
Could you use `fetch` please? There are existing examples in the codebase already.